storage/integration-test/src/storage.test.js

```
@@ -6,9 +6,9 @@ const STORAGE_URL = process.env.STORAGE_API
 const STORAGEMQ_URL = process.env.STORAGEMQ_API
 const AMQP_URL = process.env.AMQP_URL
 
-const amqp_exchange = 'ods_global'
-const amqp_pipeline_config_created_topic = 'pipeline.config.created'
-const amqp_pipeline_execution_success_topic = 'pipeline.execution.success'
+const AMQP_EXCHANGE = 'ods_global'
```

I guess you specified the values instead of using env vars because the env.ts solution is not available in the integration test, right? I think we should decide on whether we want to do it this way or use environment vars from the docker-compose file (possibly deactivate linter warnings for nonnull assertions) and keep that consistent with the other services integration tests

---

I do agree. I'm not sure, do we want failing tests if we change those parameters? Not sure..

---

Yes, maybe it's best to keep it as simple as possible, define the parameters in the test itself, and let it fail if they are changed. We should change the other integration tests so this is consistent, I will create an issue for that.